extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/ArcProxyBeanMetaDataClassNormalizer.java

```diff
@@ -6,16 +6,21 @@
 
 /**
  * In the case of a proxy generated by Arc, return the parent class.
+ * <br />
+ * This can have more than one level of hierarchy, in example:
+ * <ul>
+ * <li>When using @{@link InjectMock} or @{@link InjectSpy}</li>
```

Comment:
I don't think that this javadoc is rendered correctly. I think that you should use a FQCN here, i.e. something like `{@link io.quarkus.test.junit.mockito.InjectMock}`.